test/support/telemetry/test_helpers.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Telemetry.TestHelpers do
+  @moduledoc """
+    Provides helper methods for testing :telemetry events.
+
+    Calling start_telemetry_listener/4 registers the event_handler/1 method which echoes the telemetry event
+    back to the containing module.
+
+    This allows the use of assert_receive/3 to confirm the event was fired.
+
+    Example:
+
+    use Telemetry.TestHelpers
+
+    @event_name [:my_counter]
+
+    describe "my_event_executioner/0" do
+      test "counter event is executed", %{self: self, test: test} do
+        start_telemetry_listener(self, test, @event_name)
+
+        OUT.my_event_executioner()
+
+        assert_receive({:telemetry_event, @event_name, %{<measurement_name>: _}, _metadata})
+      end
+    end
+  """
+
+  defmacro __using__(_) do
```

Comment:
There's no reason for this to be in a using block, just call the functions normally from this module. Never use a using block without reason!!

Reply:
removed `__using__`